extensions/coap/COAPLoader.h

```diff
@@ -28,6 +28,13 @@
 #include "utils/StringUtils.h"
 #include "protocols/CoapC2Protocol.h"
 
+class COAPObjectFactoryInitializer : public core::ObjectFactoryInitializer {
+ public:
+  virtual bool initialize();
```

Comment:
Could you add override here?
I guess this PR was created before we agreed on using override.

Reply:
Done.